CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+We welcome contributions via both the raising of issues and submitting pull requests. But before you
+do, please take a moment to consult the below.
+
+## Chatting
+
+If before any of the below you'd like to discuss an issue or some code, then come and chat with
+us on one of the following platforms:
+
+* [Users mailing list](https://groups.google.com/a/apereo.org/forum/#!forum/equella-users) - Most
+  active, best for more usage centric questions
+* [Developer mailing list](https://groups.google.com/a/apereo.org/forum/#!forum/equella-dev) - For
+  discussing work on the code
+* \#Equella channel in [Apereo's slack](https://apereo.slack.com/messages/C4KLR0C4U)
+
+## Submitting Issues
+
+Issues relating to the documentation are welcome. (For any issues relating to openEQUELLA itself,
+please switch over to its repo.)
+
+When raising issues, please try and be as detailed as possible. Links, line numbers, etc always
+welcome.
+
+## Contributing Code / Pull Requests
+
+As per standard Apereo projects:  If you wish to contribute to Open EQUELLA itself, you should first
+subscribe to the Open EQUELLA Google Group as outlined above and discuss your contributions with
+the other developers. You will also need to sign a
+[Contributor License Agreement](https://www.apereo.org/node/676).
+
+But after that, as per usual GitHub, please first fork the project and then branch off `master` or a
+stable branch (see below) to use a 'feature' branch.
```

Comment:
Could you reword this? - not clear as to the meaning.  "or a stable branch (see below) to use a 'feature' branch."